ceph_bootstrap/deploy.py

```diff
@@ -1116,7 +1116,47 @@ def __init__(self, interactive):
 
         self.executor = CephSaltExecutorThread(self.controller)
 
+    @staticmethod
+    def check_ceph_salt_formula():
+        # verify that deployment can run
+        PP.println("Checking if ceph-salt formula is available...")
+        result = SaltClient.local_cmd('ceph-salt:member', 'state.sls_exists', ['ceph-salt'],
+                                      tgt_type='grain')
+        if not all(result.values()):
+            PP.println("salt-master will be restarted to load ceph-salt formula")
+            logger.info('restarting salt-master service')
+            result = SaltClient.caller_cmd('service.restart', ['ceph-salt'])
```

Comment:
Shouldn't be `salt-master` instead of `ceph-salt`?

Reply:
good catch. Pushed the fix